app/models/product.rb

```diff
@@ -1,6 +1,6 @@
 class Product < ApplicationRecord
-  has_many :order_items
-  has_many :reviews
+  has_many :order_items, dependent: :destroy
```

Comment:
If we delete :order_items when a product is destroyed, I think we'll lose the history/connection between product and order to provide a user (merchant) with total revenue. Please check my logic, but I think we'd want to change this to `dependent: :nullify`. 
```suggestion
  has_many :order_items, dependent: :nullify
```

Reply:
Makes sense Hallie, a TA actually told me the same! ;)